ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_creation.py

```diff
@@ -306,3 +308,56 @@ def test_eye_like(
         device=on_device,
         ground_truth_backend=ground_truth_backend,
     )
+
+
+@st.composite
+def _get_dtype_buffer_count_offset(draw):
+    dtype, value = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+        )
+    )
+    length = value[0].size
+    value = value[0].tobytes()
```

Comment:
You should generate 1d arrays instead of generating larger arrays and discarding the other dimensions